lib/ui/painting/image_descriptor.cc

```diff
@@ -79,13 +105,22 @@ void ImageDescriptor::initEncoded(Dart_NativeArguments args) {
 
   std::unique_ptr<SkCodec> codec =
```

Comment:
I'd add a comment here saying this call will fail if there is no built-in codec for this data and we must now try one of the fallbacks if present.